File: 4sem/java/pr5/src/Zapis.java

public class Zapis {
    // закрытые члены класса: фамилия, имя, телефон, дата рождения
    String Name_Famil;
    String Telefon;
    int[] Date = new int[3];
    int KolZap;

    // конструстор класса
    public Zapis() {

    }

    int GetColZap() {
        return KolZap;
    }

    public void SetPhoneNumber(String phoneNumber) {
        this.Telefon = phoneNumber;
    }

    String GetPhoneNumber() {
        return Telefon;
    }

    public void print() {
        System.out.print("\n" + Name_Famil + " родился " + Date[0] + "." + Date[1] + "." + Date[2]
                + ", его номер телефона: " + Telefon);
    }
}

File: java/pr8/src/main/kotlin/zapis.kt
class Zapis  // конструстор класса
    () {
    // закрытые члены класса: фамилия, имя, телефон, дата рождения
    var Name_Famil: String? = null
    var Telefon: String? = null
    var Date = IntArray(3)
    var KolZap = 0
    fun GetColZap(): Int {
        return KolZap
    }

    fun SetPhoneNumber(phoneNumber: String?) {
        Telefon = phoneNumber
    }

    fun GetPhoneNumber(): String? {
        return Telefon
    }

    fun print() {
        print(
            "\n" + Name_Famil + " родился " + Date[0] + "."  + Date[1] + "." + Date[2]
                    + ", его номер телефона: " + Telefon
        )
    }
}
